content/03.aim1.md

```diff
@@ -46,20 +48,79 @@ Simulation statistics under different parameters:
 
 Figure 1: Distribution of mean expression level of simulated genes across samples under different simulated parameters.
 
-#### 2. Visualization of simulated single cell data using VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE and PCA under different parameters
+##### 2. Visualization of simulated single cell data using VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE and PCA under different parameters
 
-To see if different methods can recover cell types, we compared the 2D visualizations on simulated datasets.
-2-layer VAE performs much better to differentiate different cell types when comparing with 1-layer VAE.
+To see if different methods can recover celltypes, we compared the 2D visualizations on simulated datasets.
+2-layer VAE performs much better to differentiate different celltypes when comparing with 1-layer VAE.
 The performance of VAE and t-SNE is similar and much better than PCA, but with the increase of outlier genes, 2-layer VAE is more resistant to noise.
 
 ![](images/outlier.vis.png)
 
 Figure 2: 2 - layer VAE is more resistant to outlier. 2D visualization of simulated single cell data from different outlier parameters ( 0 - 0.5).
 
-#### 3. Performance evaluation of different simulation parameters
+##### 3. Performance evaluation of different simulation parameters
+
 Clustering performance was measured by normalized mutual information (NMI).
 NMI is an normalization of the Mutual Information (MI, measures the dependence of two random variables). It is a measurement to determine the quality of clustering, which is between 0 (no mutual information) and 1 (perfect correlation).
 
 ![](images/nmi.png)
 
-Figure 3: Performance comparison among VAE, t-SNE and PCA under different simulation parameters. 
+Figure 3: Performance comparison among VAE, t-SNE and PCA under different simulation parameters.
+
+#### VAE performance on real single cell datasets
+
+##### 1. real single cell datasets:
+
+To evaluate the performance of VAE and the other dimension reduction approaches, we select 8 real single cell datasets with true celltype labels.
+
+| dataset | ncells | ngenes | tissue | organism | celltypes | Accesion number | Platform | celltype annotation |
+| ------- | ------ | ------ | ------ | -------- | --------- | --------- | ------------------ | -------------------------------------- |
+| yan | 90 | 20214 | Embryo Devel | human | 6 | GSE36552 | Illumina HiSeq 2000 | celltypes derived from different embryonic developmental time points |
+| goolam | 124 | 41428 | Embryo Devel | mouse | 5 | E-MTAB-3321 | | 5 celltypes derived from different embryonic developmental time points |
+| pollen | 301 | 23730 | Developing Cerebral Cortex | human | 11 | SRP041736 | Illumina HiSeq 2000 | 11 celltypes defined by different celllines |
+| patel | 430 | 5948 | Glioblastoma | human | 12 | GSE57249 | Illumina HiSeq 2000 | 5 celltypes defined from five primary glioblastomas |
```

Comment:
potential mismatch here b/w number in the final column plus number in the cell type column